app/models/reports/schema_v2.rb

```diff
@@ -126,6 +126,7 @@ def initialize(regions, periods:)
       end
     end
 
+    alias_method :missed_visits_rate, :missed_visits_rates
     alias_method :missed_visits_without_ltfu, :missed_visits
     alias_method :missed_visits_without_ltfu_rates, :missed_visits_rates
```

Comment:
`:missed_visits_rates` seems to be an alias for 2 methods?

Reply:
Yeah, I wanted to move to a plural `rates` method names, but `missed_visit_rate` is currently used in too many places to change, hence the alias here so that I can start changing it incrementally in another PR.